packages/lesmis/src/api/mutations/useLogin.js

```diff
@@ -20,9 +20,9 @@ export const useLogin = () => {
       }),
     {
       onSuccess: () => {
-        queryClient.invalidateQueries('user');
-        queryClient.invalidateQueries('entity');
-        queryClient.invalidateQueries('entities');
+        queryClient.resetQueries('user');
```

Comment:
This makes sure that the cached data against the query is not temporarily served.